examples/meta/src/evaluation/accuracy_multiclass.sg.in

```diff
@@ -0,0 +1,10 @@
+File f_labels_train = csv_file("@SHOGUN_DATA@/classifier_binary_2d_linear_labels_train.dat")
+
+#![create_labels]
+Labels labels_true = labels(f_labels_train)
+#![create_labels]
+
+#![evaluate_accuracy]
+Evaluation evaluator = evaluation("MulticlassAccuracy")
+real accuracy = evaluator.evaluate(labels_true,labels_true)
```

Comment:
this doesnt make sense, you would compute the accuracy between something that is predicted and the true thing.

Reply:
Would you prefer I use a model to predict labels or would this be way out of scope of this example?

Reply:
Do I make a model to predict labels or would this be way out of scope for this example?

Reply:
You should make the model predict the labels, otherwise the example is kind of pointless...

Reply:
Got it I will use one of the classifiers shogun has